codex/conf.nim

```diff
@@ -188,7 +188,7 @@ type
         name: "block-mn" }: int
 
       cacheSize* {.
-        desc: "The size in MiB of the block cache, 0 disables the cache - might help on slow hardrives"
+        desc: "Size in MiB of the in-RAM block cache, 0 disables the cache - might help on slow hardrives"
```

Comment:
Even though it seems NBS has not quite moved to it yet, since nim-1.6.12 dropped 3 days ago with it contained (and it is also in `stew/shims/parseutils` for older Nim), we could/should soon start using `std/parseutils.parseSize`.

The documentation for things like this is a little tricky.  It's probably best to be explicit with a suggestive example like `e.g. "512 MiB, 4G, etc."`.